azerothcore/azerothcore-wotlk
# feat(Docker): add timezone settings to dockerfile

## CHANGES PROPOSED:
-  install tzdata to Ubuntu containers
-  set time zone environment variable
-  set noninteractive environment variable (so tzdata doesnt prompt for setting timezone during install)
-  set timezone based on timezone environment variable



## ISSUES ADDRESSED:
- none



## TESTS PERFORMED:
- Docker setup (build/run) on centos 7 & checked date in running containers



## HOW TO TEST THE CHANGES:

- make sure to clear any docker cache (docker system prune -a)
- Change the time zone environment variable in the dockerfile to something other than "Etc/UTC" as UTC is the default when no timezone is set (link to list in the dockerfile)
- run a docker-compose up
- once dockers are started, run (docker exec -it "docker name" "date") 
- Verify time zone matches what you set the time zone environment variable to.

## KNOWN ISSUES AND TODO LIST:




## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
